themes/docsmith/layouts/partials/promo_code.html

```diff
@@ -5,12 +5,13 @@
                 {{ if and ($.Params.promo_code) ($.Params.promo_code_amount) }}
                 Use promo code <strong>{{ $.Params.promo_code }}</strong> for ${{ $.Params.promo_code_amount }} credit on a new account.
                 {{ else }}
-                Use promo code <strong>DOCS10</strong> for $10 credit on a new account.
-                {{ end }}
+                Sign up now using <strong> <a href="https://login.linode.com/signup?promo=DOCS20AA00X1">this link</a></strong> for $20 credit on a new account.
```

Comment:
Let's avoid using the "this link" style callout. What about "Sign up to retrieve a $20 credit on a new account"?